src/pages/loading-waiting-states/spinner-overlays/index.tsx

```diff
@@ -0,0 +1,199 @@
+        flexDirection: 'column',
+        position: 'relative',
+        padding: `${theme.spacing(2)}px ${theme.spacing(3)}px`,
+        [theme.breakpoints.down('xs')]: {
+            padding: `${theme.spacing(2)}px ${theme.spacing(2)}px ${theme.spacing(3)}px ${theme.spacing(2)}px`,
+        },
+    },
+    cardActions: {
+        padding: theme.spacing(3),
+        justifyContent: 'flex-end',
+        [theme.breakpoints.down('xs')]: {
+            padding: theme.spacing(2),
+        },
+    },
+    overlayBackground: {
+        position: 'absolute',
+        display: 'flex',
+        backgroundColor: 'rgba(255,255,255,0.6)',
+        alignItems: 'center',
+        justifyContent: 'center',
+        flexDirection: 'column',
+        top: 0,
+        left: 0,
+        width: '100%',
+        height: '100%',
+        zIndex: 4,
+    },
+}));
+
+export const SpinnerOverlays = (): JSX.Element => {
+    const dispatch = useDispatch();
+    const classes = useStyles();
+    const [eulaAccepted, setEulaAccepted] = useState(false);
+    const [eulaLoaded, setEulaLoaded] = useState(false);
+
+    const changeCheckboxState = (event: React.ChangeEvent<HTMLInputElement>): void => {
+        setEulaAccepted(event.target.checked);
+    };
+
+    useEffect(() => {
+        let timer: NodeJS.Timeout;
+        if (!eulaLoaded) {
+            timer = setTimeout(() => {
+                setEulaLoaded(true);
+            }, 3000);
+        }
+        return (): void => {
+            clearTimeout(timer);
+        };
+    }, [eulaLoaded]);
+
+    const reloadEulaDetails = useCallback((): void => {
+        setEulaLoaded(false);
+        setEulaAccepted(false);
+    }, []);
+
+    return (
+        <div style={{ minHeight: '100vh' }}>
+            <AppBar data-cy="pxb-toolbar" position={'sticky'}>
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp={true}>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <MenuIcon />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        Spinner Overlays
+                    </Typography>
+                    <div />
+                </Toolbar>
+            </AppBar>
+            <div className={classes.container}>
+                <Card className={classes.card}>
+                    {!eulaLoaded && (
+                        <div className={classes.overlayBackground}>
+                            <CircularProgress size={96} variant={'indeterminate'} color="secondary" />
```

Comment:
If you remove the color="secondary" here, that will default back to the primary blue and match angular